src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml

```diff
@@ -16,9 +16,9 @@
                                    FontFamily="Consolas"
```

Comment:
I presume these changes are for `ParameterEditor`? Wondering how that leaves `CodeBlockEditor`...